DAL/user.DAL.js

```diff
@@ -0,0 +1,42 @@
+// get all employees -- need to move ot DAL
+getAllUsers = () => {
+    return {1 : "Gilad", 2 : "Adva", 3: "Ofir", 4: "Dave"};
+};
+
+const getUserById = (id) => {
+    if(id == null)
+        return 'No valid id';
```

Comment:
if this should be an error please wrap it with an Error object and handles this as desired.